### ISSUE TITLE
Environment variables

### ISSUE THREAD

**Issue by dregert** [xvik/gradle-use-python-plugin#8]
Is there a way to set environment variables when executing a PythonTask?

**Comment by xvik** [comment]
There is no way currently, sorry. 
But it's easy to add. I will release 2.0.1 in a few days.

Just out of curiosity, could you please tell me the case when an environment variable required only for the python process and not system-wide.

**Comment by xvik** [comment]
I added support for variables but will wait for your answer (about your case) to be sure. I have some doubts that pip tasks could also need variables support (but can't figure out any usage example)

**Comment by dregert** [comment]
I need to trigger a python script (that I cannot change) and it's configured via environment variables, so I need to pass different configurations to it depending on what I'm trying to do. Therefore setting the environment variables on the system level is not really feasible. 
I did try to set the environment variables inside the gradle script (at the global level) before running a PythonTask, but if doesn't seem that the variables set are propagated to the PythonTask. 

Regarding the pip tasks, I don't have a usecase for them.

**Comment by xvik** [comment]
> I did try to set the environment variables inside the gradle script (at the global level) before running a PythonTask, but if doesn't seem that the variables set are propagated to the PythonTask.

How? It should be not possible without hacks. `Exec` task (used for python execution) should see only `System.getenv()` variables.

If you have time (and desire), you can make sure it works with the plugin [snapshot](https://github.com/xvik/gradle-use-python-plugin#snapshots) (description collapsed).

The new syntax is the same as with `Exec`:

```groovy
task sample(type: PythonTask) {
       command = "-c \"import os;print('variables: '+os.getenv('some', 'null')+' '+os.getenv('foo', 'null'))\""
       environment 'some', 1
       environment 'other', 2
       environment(['foo': 'bar', 'baz': 'bag'])
}
```

I will release plugin (hopefully) today.

**Comment by dregert** [comment]
> How? It should be not possible without hacks.

I was trying something like: `System.setProperty('FOO', 'bar')` before calling the PythonTask.

I'll try to test the snapshot later today. 
I really appreciate it! Thanks!

**Comment by dregert** [comment]
I just did a very quick test with the snapshot version and it seems to work just fine :)

**Comment by xvik** [comment]
Great! Thank you!

**Comment by xvik** [comment]
Released [2.1.0](https://github.com/xvik/gradle-use-python-plugin/releases/tag/2.1.0)
